src/teleop/gui/src/components/SelectSite.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div>
+    <div>
+      <h3>Select Site</h3>
+    </div>
+    <div class="box1">
+      <div>
+        <label for="position">Change site to position:</label>
+        <select id="site" v-model="site" name="site" @change="onChange($event)">
```

Comment:
@tabiosg Suggested we change these to radio buttons instead of drop downs and I agree.
Could be a good spot to bring in bootstrap https://getbootstrap.com/docs/5.0/components/button-group/